flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -696,6 +696,181 @@ class SqlITCase extends StreamingWithStateTestBase {
       "6,8,Hello world,51,9,5,9,1")
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
+  
+     //////////////////////////////////////////////////////
+   // START TESTING BOUNDED PROC TIME ROW AGGREGATION
+   //////////////////////////////////////////////////////
+  
+
+  @Test
+  def testSumMinPartitionedAggregatation2(): Unit = {
+
+    val env = StreamExecutionEnvironment.getExecutionEnvironment
```

Comment:
please add `env.setStateBackend(getStateBackend)` to all tests. 
This will use the RocksDBStateBackend for the tests.